index.js

```diff
@@ -4,3 +4,70 @@ console.log(square.style.left, square.style.top);
 document.addEventListener("keydown", function(e) {
   console.log(e.key)
 });
```

Comment:
As a best practice, you can remove this event listener since all your logic is in the other event listener. This function is just taking up space and memory.